### DISCUSSION TITLE
How can I have the same display of a quarto/revealjs presentation using the RStudio button and a {quarto} instruction?

### DISCUSSION THREAD

**Discussion by ddotta** [quarto-dev/quarto-cli#7069] [Q&A] [answered]
### Description

Sorry for the silly question, but I can't find the answer on the internet.  
In my quarto-revealjs presentation I don't get the same rendering of my presentation when I click on RStudio's "render" button and run a `quarto::quarto_render()`.  

My presentation is perfect only with the RStudio button....  

My guess is that when I run `quarto::quarto_render()`, it ignores the _quarto.yml file, which contains:
````qmd
project:
  title: "Présentation quarto-Masa"
  output-dir: public
  render:
    - "presentation-quarto.qmd"
    - "02_stylepardefaut.qmd"
    - "03_integrationcode.qmd"
    
format:
  masa-default-revealjs: default
  html:
    page-layout: custom
```` 

What can I do?  
Many thanks for your great work!

**Comment by mcanouil** [comment]
Use Quarto CLI without the R wrapper.
`quarto render` in a terminal (that you can open within RStudio).

**Reply by cderv** [reply]
Just to clarify for readers, running `quarto_render()` should be the same as running `quarto render`. So using R function should be fine if you need to run this from an R function. If not the same, then we have an issue to fix.

**Comment by ddotta** [comment]
It's the same problem @mcanouil. This is what I got when I use both `quarto render` and `quarto_render()`  😞 

![image](https://github.com/quarto-dev/quarto-cli/assets/13397519/0991fc08-f390-4f08-aa9e-8b4f845aac88)

**Reply by mcanouil** [reply, marked as answer]
Which format are you trying to render? Only one? Both?
If both, it is not possible without setting `output-file` because they will overwrite each other as they are both HTML.

**Reply by cderv** [reply]
@ddotta  can you share a project or a dummy project that would reproduce ? 

This would be helpful to look into. The R function and the CLI command should give same results, especially they should pick up `_quarto.yml` **except if you are not running both command from the same working directory maybe**

Thank you !

**Comment by ddotta** [comment]
@mcanouil @cderv It seems ok with only one format ! Thanks a lot for you help ! 👍
